### PR TITLE
add favicon error indicator

### PR DESCRIPTION
**Pull request by sinyo-matu**
Fixing #10

- add `favicon.ico` and `favicon-err.ico` to cli watch server resource.
- add code to handle changing favicons depending on Websocket messages.

Here are some examples
- without error
image

- with error
image

### CODE REVIEW

File: d2cli/static/watch.js [hunk 1]
```diff
@@ -73,3 +74,18 @@ function init(reconnectDelay) {
...
   };
 }
+
+const changeFavicon = function (iconURL) {
+  const head = document.getElementsByTagName("head")[0];
+  const newLink = document.createElement("link");
+  newLink.rel = "icon";
+  newLink.href = iconURL;
+  removeExistingFavicons();
+  head.appendChild(newLink);
```
Comment (alixander) [inline d2cli/static/watch.js:84]: do you have o remove then append? can you not just get the existing element and set the `href` attribute?

**Review by alixander — CHANGES_REQUESTED** [review]
thanks for this @sinyo-matu !

would you mind signing your commits please? https://github.com/terrastruct/d2/blob/master/docs/CONTRIBUTING.md#logistics

File: d2cli/static/watch.js (hunk 1 above)
Comment (sinyo-matu) [reply d2cli/static/watch.js]: @alixander sorry for the delay.
I totally agree with you.
I updated my code.
Please recheck it when you got time.

File: d2cli/watch.go [hunk 2]
```diff
@@ -429,6 +429,7 @@ func (w *watcher) handleRoot(hw http.ResponseWriter, r *http.Request) {
 	<title>%s</title>
 	<script src="./static/watch.js"></script>
 	<link rel="stylesheet" href="./static/watch.css">
+	<link id="favicon-link" rel="icon" href="./static/favicon.ico">
```
Comment (alixander) [inline d2cli/watch.go:432]: just "favicon" for the id please

**Review by alixander — CHANGES_REQUESTED** [review]
minor comment then good to go!

File: d2cli/watch.go (hunk 2 above)
Comment (sinyo-matu) [reply d2cli/watch.go]: Fixed!

**Review by alixander — APPROVED** [review]
🚀

**Pull request merged**